lib/submissions.ts

```diff
@@ -0,0 +1,73 @@
+import axios from 'axios';
+import { Submission, StepAnswers } from 'data/flexibleForms/forms.types';
+
+const { ENDPOINT_API, AWS_KEY } = process.env;
+
+const headersWithKey = {
+  'x-api-key': AWS_KEY,
+};
+
+export const startSubmission = async (
+  formId: string,
+  socialCareId: number,
+  createdBy: string
+): Promise<Submission> => {
+  const { data } = await axios.post(
+    `${ENDPOINT_API}/submissions`,
+    {
+      formId,
+      socialCareId,
+      createdBy,
+    },
+    {
+      headers: headersWithKey,
+    }
+  );
+  return data;
+};
+
+export const getSubmissionById = async (
+  submissionId: string
+): Promise<Submission> => {
+  const { data } = await axios.get(
+    `${ENDPOINT_API}/submissions/${submissionId}`,
+    {
+      headers: headersWithKey,
+    }
+  );
+  return data;
+};
+
+export const patchSubmissionForStep = async (
+  submissionId: string,
+  stepId: string,
+  editedBy: string,
+  stepAnswers: StepAnswers
+): Promise<Submission> => {
+  const { data } = await axios.patch(
+    `${ENDPOINT_API}/submissions/${submissionId}/steps/${stepId}`,
+    {
+      stepAnswers,
+      editedBy,
+    },
+    {
+      headers: headersWithKey,
+    }
+  );
+  return data;
+};
+
+export const finishSubmission = async (
+  submissionId: string
+): Promise<Submission> => {
+  const { data } = await axios.patch(
+    `${ENDPOINT_API}/submissions/${submissionId}`,
+    {
+      submittedAt: new Date(),
```

Comment:
shouldn't this be `.toISOString()`?

Reply:
I'm not sure. A test was failing because of this last week. I think you said leave the file as is and alter the test but I couldn't figure out how to do that with MockDate. It now seems to be passing fine.

Reply:
@JohnFarrellDev can say for sure. i believe we will need to submit an iso date string, so the test should represent that

Reply:
I don't think the backend will care, backend just expects a json value that can be deserialized into a DateTime object.

However, we are trying to standardise around ISO-8601 (standard date format) so I'm all for calling `.toISOString()`